app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -61,59 +56,65 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        View currView = inflater.inflate(R.layout.fragment_feed, container, false);
+        // prevent reloading if not necessary
+        if(currView == null) {
```

Comment:
Hm, not sure if this is the best way to prevent reloading -- is this what they recommended?

Reply:
I would think we'd be able to basically just navigate back to the same fragment.  It's generally fine to recreate the view, though, but I'm guessing you're doing this to save the scroll position?  If so, something you could try doing instead is just saving the scroll position in the savedInstanceState and trying to scroll to that position if it's saved in there

Reply:
I read that Navigation Component only supports replacing fragments as per [here](https://stackoverflow.com/questions/56162112/stop-reloading-previous-fragment-by-using-navigation-architecure) so I am doing that null check. As for scroll position, it isn't super important but perhaps that could help me keep the current list of shows vs. reloading a possibly entirely new set.